soc/arm/nxp_imx/rt/dts_fixup.h

```diff
@@ -41,4 +41,13 @@
 #define DT_UART_MCUX_LPUART_3_CLOCK_NAME	DT_NXP_KINETIS_LPUART_4018C000_CLOCK_CONTROLLER
 #define DT_UART_MCUX_LPUART_3_CLOCK_SUBSYS	DT_NXP_KINETIS_LPUART_4018C000_CLOCK_NAME
 
+#define DT_ETH_MCUX_0_NAME		ETH_LABEL
```

Comment:
Would prefer to see this be:

#define DT_ETH_MCUX_0_NAME		DT_NXP_KINETIS_ETHERNET_402D8000_LABEL

Reply:
@galak I've made the change you requested
@galak @pfalcon Thank you for your reviews!

Reply:
Sadly, I don't have much to be thanked for - it's on my TODO to test this patch on real hw, but unfortunately I can't to it due to other tasks I got into. I just hope that other folks tested it. (I'll get to it to of course.)

Reply:
It's taking me some time to test i.MX RT and backward checking K64F, this i why I didn't ping earlier. Patch looks okay so far.